### PR TITLE
Title Article Component complete

### PR DESCRIPTION
**Pull request by jasenscode**
- Component written
- Styling added
- Tests written for component render and props, and confirmed as passing

### CODE REVIEW

File: src/assets/components/TitleArticle/TitleArticle.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@use "../../sass/variables" as *;
+@use "../../sass/typography" as *;
+
+.title-article {
+  line-height: 150%;
+  font-family: "PT Sans";
+  &__header {
+    // font-family: $fontOne;
```
Comment (DanForder) [inline src/assets/components/TitleArticle/TitleArticle.scss:8]: remove commented code/ unused scss rules

File: src/assets/components/TitleArticle/TitleArticle.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@use "../../sass/variables" as *;
+@use "../../sass/typography" as *;
+
+.title-article {
+  line-height: 150%;
+  font-family: "PT Sans";
```
Comment (DanForder) [inline src/assets/components/TitleArticle/TitleArticle.scss:6]: create variable

File: src/assets/components/TitleArticle/TitleArticle.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { render } from "@testing-library/react";
+import TitleArticle from "./TitleArticle";
+
+test("Renders TitleArticle component", () => {
+  const { container } = render(<TitleArticle />);
```
Comment (DanForder) [inline src/assets/components/TitleArticle/TitleArticle.test.js:5]: use customRender function

**Review by DanForder — CHANGES_REQUESTED** [review]

**Comment by DanForder** [comment]
You have conflicts with app.jsx, make sure you pull from main and handle any conflicts

File: src/components/ViewScheduleButton/ViewScheduleButton.css [hunk 4]
```diff
@@ -0,0 +1 @@
+@import"https://fonts.googleapis.com/css2?family=PT+Sans&family=Poppins:wght@400;500;600;700&display=swap";@import"https://fonts.googleapis.com/css2?family=Lato&family=Playfair+Display:wght@400;500;600;700&display=swap";.view-schedule-button{height:66px;width:335px;text-transform:uppercase;background-color:#3c3c3c;border-radius:5px;color:#fff;font-weight:700;font-family:"Poppins",sans-serif;font-size:1rem;display:flex;flex-direction:column;justify-content:center;align-items:center;text-decoration:none}
```
Comment (DanForder) [inline src/components/ViewScheduleButton/ViewScheduleButton.css:1]: remove this file

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/ViewScheduleButton/ViewScheduleButton.css (hunk 4 above)
Comment (jasenscode) [reply src/components/ViewScheduleButton/ViewScheduleButton.css]: Removed

**Review by DanForder — APPROVED** [review]

**Pull request merged**